code/ui-utils/src/main/java/com/adobe/marketing/mobile/utils/UiImageConstants.kt

```diff
@@ -0,0 +1,15 @@
+package com.adobe.marketing.mobile.utils
+
+import java.util.concurrent.TimeUnit
+
+object UiImageConstants {
+    const val DEFAULT_BITMAP_QUALITY = 100
+    const val DEFAULT_DOWNLOAD_TIMEOUT_SECS = 10
+    const val CAROUSEL_MAX_BITMAP_WIDTH = 300
+    const val CAROUSEL_MAX_BITMAP_HEIGHT = 200
+    const val PUSH_TEMPLATE_LOG_TAG = "PushTemplates"
+    const val CACHE_BASE_DIR = "pushtemplates"
```

Comment:
The UI utils library is generic to UI libraries. Lets use a different log tag. UI-`UI_UTIL_LOG_TAG =UIUtils`